airflow/providers/amazon/aws/operators/redshift_resume_cluster.py

```diff
@@ -0,0 +1,57 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.models import BaseOperator
+from airflow.providers.amazon.aws.hooks.redshift import RedshiftClusterStates, RedshiftHook
+
+
+class RedshiftResumeClusterOperator(BaseOperator):
```

Comment:
This is just my opinion but I think that the AWS provider has too many files.
I think ideally operators for the same service should be together in the same file - Just like the GCP provider.
E.g 
we shouldn't have `redfshift_result_cluster.py` & `redfshift_result_cluster.py` but we should have `redfshift.py` that contains the 2 operator classes.

Reply:
Thanks @eladkal! Should i move the 2 operators to `redfshift.py` then?

Reply:
@dstandish WDYT?
I just love how the GCP is organized with file per service https://github.com/apache/airflow/tree/main/airflow/providers/google/cloud/operators

Reply:
For what is worth, i do think it's kind of crazy to have a "file-per-operator". If my vote counts for anything I would prefer to have the operators to `redshift.py`

Reply:
+1 for a file per service

Reply:
For what it's worth, I also considered pointing that out and I agree.  One file per service (well.. one file in hooks, one in operators, one sensors, etc) should be the way to go IMHO.

`/airflow/providers/amazon/aws/operators/service_name.py` would make it easy to find what you are looking for.

Reply:
no objection from me

however i will defend having separate modules when it makse sense.  if a service has a _lot_ of operators, or very long and complicated operators, or  certain kinds or families of operators for the same service, why not split into a few modules 🤷

Reply:
In the next few days ill open an issue with full mapping of the aws operators and we can discuss each case.

For this specific PR it appears that we are all agreed so @dbarrundiag you can proceed with single file :)

Reply:
Followup issue https://github.com/apache/airflow/issues/20139